pkg/config/config.go

```diff
@@ -137,6 +137,10 @@ type ContainersConfig struct {
 	// Negative values indicate that the log file won't be truncated.
 	LogSizeMax int64 `toml:"log_size_max,omitempty"`
 
+	// Specifies default format tag for container log messages.  Helpful for specific
+	// container log, messages. Containers logs default to truncated container ID.
```

Comment:
If you change the above, change here too.

Reply:
Fixed

Reply:
Something must have gone wrong while copy-pasting.  The sentence above starts in the middle.

Reply:
Weird,  Fixed.